context-linux/src/etc/one-context.d/net-99-report-ready

```diff
@@ -69,14 +69,29 @@ while [ "$RETRY_COUNT" -gt 0 ] ; do
             fi
             ;;
     esac
+}
+
+while [ "$RETRY_COUNT" -gt 0 ] ; do
+    if [[ -n "$READY_SCRIPT" ]];then
+        if eval "$READY_SCRIPT";then
```

Comment:
I'd suggest spawning a subshell here, just like:

```suggestion
        if (eval "$READY_SCRIPT"); then
```

That way if somebody includes an `exit` command it will exit only the subshell and not the whole script.